db/migrate/20210504151301_remove_index_models_on_record_type_and_record_id.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveIndexModelsOnRecordTypeAndRecordId < ActiveRecord::Migration[6.1]
+  def change
+    remove_index :models, name: "index_models_on_record_type_and_record_id" , if_exists: true
+  end
+end
```

Comment:
You can merge both migrations into one

Reply:
I thought the same but was not sure. So created two migrations. will merge both :)